automation/automation/jsenv_handler.yaml

```diff
@@ -0,0 +1,25 @@
+imports:
+  - io
+
+functions:
+  process:
+    meta:
+      short: Process arbitrary data in jsenv
+    params:
+      scope:
+        required: true
+        types:
+          - { wf: Any }
+          - { wf: String, suffix: String }
+          - { wf: Reader, suffix: Stream }
+      function:
```

Comment:
would "source" be a better name for this here?

Reply:
it would, thanks